Bottom: 247933c364d02747ae5c0bc8f66f31471692e694
Top:    f0b70e7842dbfb8c6a8f6fb51b5abdfd81dd0a42
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:07 +0000

Subtract vias from ...


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c19a0ba..e252ff1 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1296,6 +1296,10 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
        *
        */
 
+      fprintf (stderr, "Subtracting pin + via drills\n");
+      r_search (PCB->Data->pin_tree, &bounds, NULL, pv_drill_callback, &info);
+      r_search (PCB->Data->via_tree, &bounds, NULL, pv_drill_callback, &info);
+
       if (info.poly == NULL)
         {
           fprintf (stderr, "Skipping layer group %i, info.poly was NULL\n", group);
